### DISCUSSION TITLE
feat: move the Connections, Models and Images sections to the admin panel settings

### DISCUSSION THREAD

**Discussion by Lanhild** [open-webui/open-webui#2865] [General]
The `Connections`, `Models` and `Images` sections of the Settings form all are actions available to admins only. Therefore, they should be in the admin panel settings instead.

![image](https://github.com/open-webui/open-webui/assets/73610779/03e173d5-3319-4915-befa-c6a7cd6c1edd)

**Comment by tjbck** [comment]
Connections, Models should stay for ease of access, I agree with Images section. Feel free to make a PR.
